src/main/java/seedu/kitchenhelper/command/AddIngredientCommand.java

```diff
@@ -128,8 +157,22 @@ public void sortIngredientList(ArrayList<Ingredient> ingredientList) {
     @Override
     public CommandResult execute(ArrayList<Ingredient> ingredientList, ArrayList<Recipe> recipeList,
                                  ArrayList<Chore> choreList) {
-        addToCategory(categoryName, ingredientList);
-        kitchenLogs.info(logAddIngredient);
-        return new CommandResult(String.format(MESSAGE_SUCCESS, ingredientName, categoryName, quantity, price, expiry));
+        int index = mergeTogetherIndex(ingredientList);
+        
+        if (index != NEGATIVE_ONE) {
```

Comment:
Not quite sure what `NEGATIVE_ONE` is 😕 Maybe you meant `indexNotFound`?

Reply:
Just -1. don't want use magic numbers.